docs/_future/create_new_app.md

```diff
@@ -0,0 +1,765 @@
+
+```js
+TimeOffWorkflow.addStep(ApprovalFunction, {
+  employee: TimeOffWorkflow.inputs.interactivity.interactor.id,
+  manager: step1.outputs.fields.manager,
+  start_date: step1.outputs.fields.start_date,
+  end_date: step1.outputs.fields.end_date,
+});
+```
+
+This step passes in a custom `ApprovalFunction` as well as inputs from the form in the first workflow step: `manager`, `start_date`, and `end_date`. It also passes in the `employee` ID by using the `interactivity` property's `interactor` ID, which comes from the initial definition of the `TimeOffWorkflow`.
+
+Once the function step is executed, that will be the end of the workflow! All remaining functionality, such as notifying the requester whether their request was approved or denied, will be executed using the `ApprovalFunction` and its related action handlers. The `ApprovalFunction` definition and implementation will be covered in the [Functions](#functions) section below.
+
+The final workflow file will look like this:
+```js
+// /manifest/workflows/time-off-request.js
+const { DefineWorkflow, Schema } = require('@slack/bolt');
+// this function will live in manifest/functions/approval.js
+const { ApprovalFunction } = require('../functions/approval');
+
+const TimeOffWorkflow = DefineWorkflow({
+  callback_id: 'time_off_request_wf',
+  title: 'Time Off Request Workflow',
+  input_parameters: {
+    properties: {
+      interactivity: {
+        type: Schema.slack.types.interactivity,
+      },
+    },
+    required: [],
+  },
+});
+
+const step1 = TimeOffWorkflow.addStep(
+  Schema.slack.functions.OpenForm,
+  {
+    title: 'Request Time Off',
+    submit_label: 'Request',
+    description: 'Please describe your request',
+    interactivity: TimeOffWorkflow.inputs.interactivity,
+    fields: {
+      required: ['manager', 'start_date', 'end_date'],
+      elements: [
+        {
+          name: 'manager',
+          title: 'Manager',
+          type: Schema.slack.types.user_id,
+        },
+        {
+          name: 'start_date',
+          title: 'Start Date',
+          type: Schema.slack.types.timestamp,
+        },
+        {
+          name: 'end_date',
+          title: 'End Date',
+          type: Schema.slack.types.timestamp,
+        },
+      ],
+    },
+  },
+);
+
+TimeOffWorkflow.addStep(ApprovalFunction, {
+  employee: TimeOffWorkflow.inputs.interactivity.interactor.id,
+  manager: step1.outputs.fields.manager,
+  start_date: step1.outputs.fields.start_date,
+  end_date: step1.outputs.fields.end_date,
+});
+
+module.exports = { TimeOffWorkflow };
+```
+
+To explore more about Workflows and dive into creating new ones, visit the guide [here](/bolt-js/future/workflows).
+
+---
+### Triggers {#triggers}
```

Comment:
```suggestion
### Link Triggers {#triggers}
```
This is a link trigger, and maybe we send users to the more general page on triggers?

Reply:
sounds good! I'll keep in the blurb about link triggers at the top and then cut out everything up to "To learn more about Triggers and create new ones, visit the guide [here]"